File: fixtures/elixir/get_basic_auth.ex
request = %HTTPoison.Request{
  method: :get,
  url: "https://api.test.com/",
  options: [hackney: [basic_auth: {~s|some_username|, ~s|some_password|}]],
  headers: [],
  params: [],
  body: ""
}

response = HTTPoison.request(request)

File: test/fixtures/node-request/get_basic_auth.js
var request = require('request');

var options = {
    url: 'http://localhost:28139/',
    auth: {
        'user': 'some_username',
        'pass': 'some_password'
    }
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);
